list_of_subprojects.md

```diff
@@ -0,0 +1,66 @@
+
+- nbdime (https://github.com/jupyter/nbdime)
+- nbgrader (https://github.com/jupyter/nbgrader)
+- QtConsole (https://github.com/jupyter/qtconsole)
+- ipyparallel (https://github.com/ipython/ipyparallel)
+- All other repos not listed above and that don't clearly fall into one of the official Subproject GitHub organizations above.
+
+## Notes about Official Jupyter Subprojects
+
+The Official Jupyter Subprojects document proposes a number of changes to how our repositories are organized into official Subprojects and GitHub organizations. This document describes the changes that are being proposed.
+
+- Services
+  - In general Jupyter services such as Binder, nbviewer, and the Jupyter website involve legal, financial, and operational matters. As such, we are proposing that the actual services are managed by working groups that report to the Board of Directors, who can provide the needed support and oversight. For example, if we want to hire full or part time staff to maintain or operate these services, the Board of Directors would be responsible for raising funds, hiring, and managing those staff.
+  - The JupyterHub and Binder teams have a number of unique considerations. Currently all Binder repositories are under the Jupyterhub organizations, but there is a separate (but highly overlapping team) listed as the Binder team. These teams will need to work out if they would like to each have a formal decision making body with an SSC representative, or if they would like to have a single team. The Governance Working Group will be available to the JupyterHub and Binder teams to help working through these questions.
+  - We propose that the Jupyer website and its repository will be governed by a working group that reports to the Board of Directors.
+  - We propose that the nbviewer service (only the actual live service) be governed by a working group that reports to the Board of Directors. The reusable part of nbviewer contained (https://github.com/jupyter/nbviewer) will be governed by the Jupyter Enterprise Subproject.
+- Jupyter Kernels
+  - To consolidate the project’s work on first-party kernels, we propose to establish a new official Subproject called _Jupyter Kernels_ and create a Github organization named jupyter-kernels for the work of the Subproject. All Xeus repositories (https://github.com/jupyter-xeus) will be moved into this organization. This Subproject will also govern the IPython GitHub organization, which will be left in place (https://github.com/ipython).
+  - A new decision making body for this organization will be established, and they will elect an SSC delegate.
+- Jupyter Standards
```

Comment:
What we've had in mind is that the entire SSC is the body responsible for discussing/voting on JEPs, so that a JEP's impact on any number of individual components can be assessed.

The management of the JEP repo and stewardship is indeed left unstated here - I think putting it under Standards makes a lot of sense, with the above consideration on decisions/voting.

Will update the PR as per this, thx! Obviously let us know if you see more to discuss :)

Reply:
I think it makes sense to put the JEP process under the standards group, given that overseeing this process might require a specialized set of skills and a particular interest in community decision making, which seems overlapping with what the standards group will need to have